resources/config.edn

```diff
@@ -2,4 +2,6 @@
  :nrepl {:enable? #boolean #or [#env ENABLE_NREPL "true"]
          :port #long #or [#env NREPL_PORT 4444]}
  :secrets #include "secrets.edn"
- :database {:jdbc-url "jdbc:sqlite:reviews_next.db"}}
+ :database
+        {:jdbc-url "jdbc:sqlite:reviews_next.db"
+         :test "jdbc:sqlite:test_reviews_next.db"}}
```

Comment:
We can change this so that the database url is injected from the environment. 
Eg: ` :database {:jdbc-url #env DATABASE_URL}`